src/main/java/com/northconcepts/datapipeline/examples/cookbook/ParseInvalidExpressions.java

```diff
@@ -0,0 +1,60 @@
+package com.northconcepts.datapipeline.examples.cookbook;
+
+import com.northconcepts.datapipeline.internal.expression.Expression;
+import com.northconcepts.datapipeline.internal.expression.InvalidSyntax;
+import com.northconcepts.datapipeline.internal.parser.Parser;
+
+public class ParseInvalidExpressions {
+
+    public static void main(String[] args) {
+        parseAnInvalidExpression();
+        parseInvalidExpressions();
+    }
+
+    private static void parseAnInvalidExpression() {
+        System.out.println("=========================== Parse An Invalid Expression =======================================");
+        // Parse an invalid expression which is missing closing parenthesis
+
+        Expression expression = Parser.parseExpression("toInt('123'", true);
+
+        InvalidSyntax invalidSyntax = (InvalidSyntax) expression;
```

Comment:
Let's extract this code as a separate method (`executeExpression(Expression expression)`) for use here and with the 2 expressions below.

Reply:
Created this common method.